pkg/graph/resolvers/operational_need_triggers_test.go

```diff
@@ -1,223 +1,223 @@
-	suite.NoError(err)
-
-	manageProvOverlap := findOpNeed(opNeeds, models.OpNKManageProvOverlap)
-	suite.NotNil(manageProvOverlap)
-	suite.Nil(manageProvOverlap.Needed)
-
-	changes["providerOverlap"] = string(models.OverlapNo) //not needed
-	updatedPP, err = PlanParticipantsAndProvidersUpdate(suite.testConfigs.Logger, pp.ID, changes, suite.testConfigs.Principal, suite.testConfigs.Store)
-	suite.NotNil(updatedPP)
-	suite.NoError(err)
-
-	opNeeds, err = OperationalNeedCollectionGetByModelPlanID(suite.testConfigs.Logger, plan.ID, suite.testConfigs.Store)
-	suite.NoError(err)
-
-	manageProvOverlap = findOpNeed(opNeeds, models.OpNKManageProvOverlap)
-	suite.NotNil(manageProvOverlap)
-	suite.False(*manageProvOverlap.Needed)
-
-	changes["providerOverlap"] = string(models.OverlapYesNeedPolicies) //needed
-	updatedPP, err = PlanParticipantsAndProvidersUpdate(suite.testConfigs.Logger, pp.ID, changes, suite.testConfigs.Principal, suite.testConfigs.Store)
-	suite.NotNil(updatedPP)
-	suite.NoError(err)
-
-	opNeeds, err = OperationalNeedCollectionGetByModelPlanID(suite.testConfigs.Logger, plan.ID, suite.testConfigs.Store)
-	suite.NoError(err)
-
-	manageProvOverlap = findOpNeed(opNeeds, models.OpNKManageProvOverlap)
-	suite.NotNil(manageProvOverlap)
-	suite.True(*manageProvOverlap.Needed)
-
-	changes["providerOverlap"] = string(models.OverlapYesNoIssues) // still needed with diff answer
-	updatedPP, err = PlanParticipantsAndProvidersUpdate(suite.testConfigs.Logger, pp.ID, changes, suite.testConfigs.Principal, suite.testConfigs.Store)
-	suite.NotNil(updatedPP)
-	suite.NoError(err)
-
-	opNeeds, err = OperationalNeedCollectionGetByModelPlanID(suite.testConfigs.Logger, plan.ID, suite.testConfigs.Store)
-	suite.NoError(err)
-
-	manageProvOverlap = findOpNeed(opNeeds, models.OpNKManageProvOverlap)
-	suite.NotNil(manageProvOverlap)
-	suite.True(*manageProvOverlap.Needed)
-
-	//multi-select test
-	changes["selectionMethod"] = []string{model.ParticipantSelectionTypeApplicationReviewAndScoringTool.String(), model.ParticipantSelectionTypeApplicationSupportContractor.String()}
-	updatedPP, err = PlanParticipantsAndProvidersUpdate(suite.testConfigs.Logger, pp.ID, changes, suite.testConfigs.Principal, suite.testConfigs.Store)
-	suite.NotNil(updatedPP)
-	suite.NoError(err)
-
-	opNeeds, err = OperationalNeedCollectionGetByModelPlanID(suite.testConfigs.Logger, plan.ID, suite.testConfigs.Store)
-	suite.NoError(err)
-	colRevScoreApp := findOpNeed(opNeeds, models.OpNKColRevScoreApp)
-	suite.NotNil(colRevScoreApp)
-	appSuppCont := findOpNeed(opNeeds, models.OpNKAppSupportCon)
-	suite.NotNil(appSuppCont)
-
-	suite.True(*colRevScoreApp.Needed)
-	suite.True(*appSuppCont.Needed)
-
-}
+// import (
+// 	"reflect"
+
+// 	"github.com/cmsgov/mint-app/pkg/graph/model"
+// 	"github.com/cmsgov/mint-app/pkg/models"
+// )
+
+// func (suite *ResolverSuite) TestGeneralCharacteristicsNeeds() {
```

Comment:
I think what might work well here is calling something like this at the start of each test:

```go
suite.T().Skip("TODO: Skipping while operational needs trigger is disabled")
```

This way you don't get out-of-date testing code if something changes related to a type/method used in our tests, and you get visibility in our testing suite as to the fact that tests exist that are being skipped (as opposed to comments, which the testing suite has no clue they even exist)

Reply:
From local testing, this might also require a lint ignore on `findOpNeed` (the linter is smart enough to know that `t.Skip()` makes the following code unreachable!

```go
//lint:ignore U1000 We are currently calling `t.Skip()` on tests, and this method is unused because of it.
func findOpNeed(collection []*models.OperationalNeed, key models.OperationalNeedKey) *models.OperationalNeed {
...
```

Reply:
Thanks Clay! That is a good call, I'll implement it.